## Calibration frame pose
# This file specifies the poses of the frames used in the calibration.

# Calibration frame position (Relative to panda_link0 frame)
calib_frame_x_pos: 0.3  # Relative position on the x axis [m]
calib_frame_y_pos: -0.03 # Relative position on the y axis [m]
calib_frame_z_pos: 0.015 # Relative position on the z axis [m]
calib_frame_yaw: 0.0                  # Rotation around the z axis [rad]
calib_frame_pitch: 0.0               # Rotation around the y axis [rad]
calib_frame_roll: 0.0                # Rotation around the x axis [rad]

# Sensor start frame position (Relative to calibration frame) #
sensor_frame_x_pos: 0.052253043575496975 # Relative position on the x axis [m]
sensor_frame_y_pos: -0.4808043350026767   # Relative position on the y axis [m]
sensor_frame_z_pos: 0.6605327857108133  # Relative position on the z axis [m]
sensor_frame_q1: 0.232811053868441     # 1st orientation quaternion [rad]
sensor_frame_q2: -0.1981809357311739  # 2th orientation quaternion [rad]
sensor_frame_q3: -0.7132844472141844    # 3th orientation quaternion [rad]
sensor_frame_q4: -0.6306731540756572     # 4th orientation quaternion [rad]
sensor_frame_yaw: -2.1953878428371025 # Rotation around the z axis [rad]
sensor_frame_pitch: -0.0058491663508139675 # Rotation around the y axis [rad]
sensor_frame_roll: 0.1092182112432458      # Rotation around the x axis [rad]

